package.json

```diff
@@ -41,21 +44,15 @@
       "react-app/jest"
     ]
   },
-  "browserslist": {
-    "production": [
+  "browserslist": [
       ">0.2%",
       "not dead",
       "not op_mini all"
-    ],
-    "development": [
-      "last 1 chrome version",
-      "last 1 firefox version",
-      "last 1 safari version"
-    ]
```

Comment:
Not that I think it affects anything, but does changing this do anything?

Reply:
as above, gonna check it out.